applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -0,0 +1,164 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config BT_AUDIO
+	bool
+	default y
+
+# Not supported on Packetcraft controller
```

Comment:
Probably we have to replace Packetcraft to "LE Audio Controller Subsystem for nRF53"

Reply:
Yes, good catch

Reply:
Done